src/test/rimgaile/FibonacciTest.java

```diff
@@ -27,8 +27,19 @@ public void indexIsValid () {
     public void indexInvalid () {
         Assert.assertEquals(false, Fibonacci.validIndex(95));
     }
+
     @Test
-    public void indexIsEdge () {
+    public void indexIsMaxEdge () {
         Assert.assertEquals(false, Fibonacci.validIndex(94));
```

Comment:
You can use Assert.assertFalse(...) instead of Assert.assertEquals(false, ...)

Reply:
Thanks! Fixed